### ISSUE TITLE
Add customizations from DSpace 5 dspace.cfg, modules/cfg, and build.properties to local.cfg

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#592]
Add customizations from DSpace 5 dspace.cfg, modules/cfg, and build.properties and Ansible templates to local.cfg.

**Comment by alawvt** [comment]
in dspace.cfg (DSpace 6)
- db.maxidle = 10 

in build.properties (DSpace 5)
- db.maxidle = -1
- db.statementpool = true
- db.poolname = dspacepool

in build.properties (DSpace 5)
- handle.canonical.prefix = http://hdl.handle.net/

in local.cfg
- mail.from.address = no-reply@vtechworks.lib.vt.edu
- feedback.recipient = vtechworks@vt.edu
- mail.admin = vtechworks@vt.edu
- registration.notify = ${mail.admin}

in build.properties (DSpace 5)
- mail.server = smtp.example.com
- mail.from.address = system@vtechworks.lib.vt.edu
- mail.feedback.recipient = vtechworks@vt.edu
- mail.alert.recipient = vtechworks@vt.edu
- mail.registration.notify = vtechworks@vt.edu

**Comment by alawvt** [comment]
dspace.cfg (DSpace 5) vs dspace.cfg (DSpace 6)

dspace.cfg (DSpace 6)
##### Email settings ######
mail.server = smtp.example.com
mail.server.username =
mail.server.password =
mail.server.port = 25
mail.from.address = dspace-noreply@myu.edu
feedback.recipient = dspace-help@myu.edu
mail.admin = dspace-help@myu.edu
alert.recipient = ${mail.admin}

dspace.cfg (DSpace 5) with overwritten settings from build.properties
##### Email settings ######
mail.server = ${mail.server}
mail.server.username = ${mail.server.username}
mail.server.password = ${mail.server.password}
mail.server.port = ${mail.server.port}
mail.from.address = ${mail.from.address} = system@vtechworks.lib.vt.edu
feedback.recipient = ${mail.feedback.recipient} = vtechworks@vt.edu
mail.admin = ${mail.admin}
alert.recipient = ${mail.alert.recipient} = vtechworks.vt.edu
registration.notify = ${mail.registration.notify}

**Comment by alawvt** [comment]
I have reorganized local.cfg into 3 sections:

```

##########################################################################################
# VTechWorks Configuration Organization                                                  #
#                                                                                        #
# This file is organized with sections:                                                  #
#                                                                                        #
# Overwritten by Ansible:   Variables overwritten by vtlibans site_secrets.yml variables.#
#                                                                                        #
# Not overwritten by Ansible: Variables in dspace.cfg that are overwritten in local.cfg  #
#                             in roughly the order the variables appear in dspace.cfg.   #
#                                                                                        #
# Overwritten modules files:  Variable in config/modules files overwritten by local.cfg, #
#                             listed in alphebetical order of module filenames.          #
#                                                                                        #
# Note: config/authentication-ip.cfg has been customized directly.                       #
########################################################################################## 
```

Please make changes in the appropriate section. I am testing it in branch, `organize_local_cfg`.

**Comment by keithgee** [comment]
I love these comments about the file organization.

**Comment by alawvt** [comment]
Thanks, @keithgee!

**Comment by alawvt** [comment]
It looks like changes for `dspace/config/modules/authentication-ip.cfg` and `dspace/config/modules/authentication.cfg` were made directly to those files for DSpace 6. I have moved the changes for the variables in `authentication.cfg` to `local.cf`g and left the changes for variables in `authentication-ip.cfg`.

**Comment by alawvt** [comment]
Resolved with PR #608.
